.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,50 @@
+name: Publish to PyPi
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    name: Build distribution 📦
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.11"
+    - name: Install pypa/build
+      run: >-
+        python3 -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: python3 -m build
+    - name: Store the distribution packages
+      uses: actions/upload-artifact@v3
+      with:
+        name: python-package-distributions
+        path: dist/
+
+  publish-to-pypi:
+    name: >-
+      Publish Python 🐍 distribution 📦 to PyPI
+    if: startsWith(github.ref, 'refs/tags/')  # only publish to PyPI on tag pushes
```

Comment:
Maybe we'd need to add 

```
on:
  push:
    tags:
      # Don't try to be smart about PEP 440 compliance,
      # see https://www.python.org/dev/peps/pep-0440/#appendix-b-parsing-version-strings-with-regular-expressions
      - v*
```

on top? Otherwise even if the `if: startsWith(...` check is there, this will never get triggered

See https://github.com/astrojuanlu/copier-pylib/blob/main/template/.github/workflows/publish.yml.j2

Reply:
An alternative flow (which I linked in my comment above) is publishing to PyPi on releases:

```
on:
  release:
    types: [created]
```

I like this approach as publishing to PyPi is not triggered by changes in the repo, rather it only happens when we purposefully want to release a new version.

Thoughts?

Reply:
Looks good to me!

Reply:
Done in [3eda138](https://github.com/poliastro/czml3/pull/138/commits/3eda1382ee067177f1255e24497fab177f46095e)